README.md

```diff
@@ -8,13 +8,15 @@ This is where you’ll find out How We Do Frontend Things at Bitcrowd™.
 
 
 ## Tools and Libraries
+
 Tools and libraries used regularly at bitcrowd.
 
-# Basic frontend tools
+## Basic frontend tools
+
 * [Homebrew](http://brew.sh/) - Used on OS X to install almost everything else.
 * [nvm (node version manager)](https://github.com/creationix/nvm) - Enables use of multiple Node.js versions.
 * [Rbenv](https://github.com/rbenv/rbenv) - Enables use of multiple versions of Ruby.
 * [npm](https://www.npmjs.com/) - Package manager for Node.js.
-* [Gulp](http://gulpjs.com/) - Javascript build and automation system.
-* [Grunt](http://gruntjs.com/) - Frontend javascript task runner.
+* [Gulp](http://gulpjs.com/) - JavaScript build and automation system.
+* [Grunt](http://gruntjs.com/) - Frontend JavaScript task runner.
```

Comment:
We can probably just remove these — they fell out of use years ago :D wdyt?

Reply:
Yep. Should bitstyles also be listed here somewhere?

Reply:
Maybe, tho it doesn’t really fit with the rest of these tools — they’re more like build tools. If we’re including the frameworks etc we use, that should maybe include react, storybook etc.